test/gemm/gemm_kernel_base_impl.hpp

```diff
@@ -835,16 +835,29 @@ namespace rocwmma
                         LayoutA,
                         LayoutB,
                         LayoutC,
-                        LayoutD>::reportResults()
+                        LayoutD>::reportResults(std::ostream& stream,
+                                                bool isFstream,
+                                                bool omitSkipped,
+                                                bool omitFailed,
+                                                bool omitPassed)
     {
-
-        if(!KernelI::sHeaderPrinted)
+        // Print header to std::cout
```

Comment:
refer to new boolean here if we will print header.
Don't refer to the static variable anymore.